next.constants.mjs

```diff
@@ -25,7 +25,7 @@ export const VERCEL_REVALIDATE = Number(
 );
 
 /**
- * This is used for telling Next.js to to a Static Export Build of the Website
+ * This is used for telling Next.js to a Static Export Build of the Website
```

Comment:
```suggestion
 * This is used for telling Next.js to do a Static Export Build of the Website
```
I think this might have been a typo where the intention was the word `do`